BLAS/SRC/CMakeLists.txt

```diff
@@ -105,3 +105,11 @@ set_target_properties(
   SOVERSION ${LAPACK_MAJOR_VERSION}
   )
 lapack_install_library(${BLASLIB})
+if(BUILD_SHARED_LIBS AND BUILD_STATIC_LIBS)
+  add_library(${BLASLIB}_static STATIC ${SOURCES})
```

Comment:
Does lapack export those targets as `-config.cmake?` If it does please provide a non-suffixed alias which can be switched between static/shared

Reply:
I don't know enough CMake to follow through on that. An alias for what exactly? I must admit that I'd like a world (on Unix-y systems, yes) with more pkgconfig files and less specific CMake build config  files. I have no contact to those if I can help it:-/

Reply:
> with more pkgconfig files 

proper pkgconfig is difficult and mixing shared/static is in my eyes bad. vcpkg has corrective measures to fix pc files depending of the specified lib linkage and als makes them relocatable by using `${pcfiledir}`

As far as i can tell lapack exports its targets since this change only affects builds of both libraries I don't care that much since vcpkg always builds either shared or static.  

> I have no contact to those if I can help it

People should just have a consistent interface in using lapack targets. As long as the single lib build is not affect this is not an issue. Only if people have both libs they need to decide to use `lapack_static` or `lapack` which is annoying for downstream users. as such you would normally add an imported interface targets (not an alias because those are sometimes problematic) which depending on some user setting switches between `lapack_static` and `lapack_shared`.

Reply:
I second my comment below about rather getting rid of this Cmake-specific dependency mechanism. But, after all this time, I feel indeed that I could make my life easier by dropping everything about static libraries here. I wanted to support them out of tradition. They used to be provided as a default. But if we have discussions spanning years about building them alongside or not … maybe it's finally time to get rid of them also in our system-wide installs. I could re-visit my branch and rip out the static-and-shared-at-the-same-time stuff and see if the other changes could be massaged to be acceptable. I'd need to convince folks over at pkgsrc that nobody really needs static libs anymore. I guess that should work.